web-common/src/features/dashboards/stores/filter-utils.ts

```diff
@@ -386,3 +390,34 @@ export function isExpressionUnsupported(expression: V1Expression) {
 
   return false;
 }
+
+export function getValidFilterForMetricView(
```

Comment:
A few things:
1. Nit: for consistency, "MetricView" -> "MetricsView"
2. Given this function's name, I'd expect it to take a `metricsView` as a parameter
3. `compose` might be a more descriptive verb? As is, the function could use a comment to explain what it actually does.